[11-Nov-2018 16:11:39 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'mysqli::__construct(): (HY000/1049): Unknown database 'a''
Stack trace: #0 [internal function]: Core\Error::errorHandler(2, 'mysqli::__const...', 'C:\\xampp\\htdocs...', 23, Array)
#1 C:\xampp\htdocs\belajar_php\public\index.php(23): mysqli->__construct('localhost:3306', 'root', '', 'a')
#2 {main}
Thrown in 'C:\xampp\htdocs\belajar_php\public\index.php' on line 23
